aspnetcore/web-api/index.md

```diff
@@ -222,6 +223,14 @@ When an action has more than one parameter bound from the request body, an excep
   public IActionResult Action3([FromBody] Product product, [FromBody] Order order)
   ```
 
+### FromServices inference notes
+
+`[FromServices]` inference could, in rare cases, affect the application if a parameter type is registered in the Dependency Injection container and also expected to be bound from a different source, eg. `FromBody`.
+
+In those cases, is possible to avoid the inference adding the desired binding source attribute to the Action parameter, or disabling the `[FromServices]` inference rule, setting <xref:Microsoft.AspNetCore.Mvc.ApiBehaviorOptions.DisableImplicitFromServicesParameters> to `true` in `Startup.ConfigureServices`:
```

Comment:
The Microsoft.AspNetCore.Mvc.ApiBehaviorOptions.DisableImplicitFromServicesParameters will be only in .NET 7 and I getting, as expected, a warning during the publishing. 

![image](https://user-images.githubusercontent.com/12376385/155236139-c469c90f-e0d5-48f8-b615-26a2b07793ef.png)

@Rick-Anderson, how we usually handle this?

Reply:
@brunolins16  hold on, I'll fix it with monikers so it will only show up in the .NET 7 version.

Reply:
I'll fix it. That API isn't published yet.